src/uu/chmod/chmod.md

```diff
@@ -0,0 +1,14 @@
+# chmod
+
+```
+chmod [OPTION]... MODE[,MODE]... FILE...
+chmod [OPTION]... OCTAL-MODE FILE...
+chmod [OPTION]... --reference=RFILE FILE...
+```
+
+Change the mode of each FILE to MODE.
+With --reference, change the mode of each FILE to that of RFILE.
+
+## Long Usage
```

Comment:
Could you rename this to "After help" here and in the code for consistency with the other utils?